service/src/test/java/tds/student/services/impl/RtsServiceImplTest.java

```diff
@@ -37,7 +38,7 @@ public void setUp() {
     public void shouldReturnEmptyIfPackageCannotBeFound() {
```

Comment:
Possibly add tests for passing in `null` or empty array (whatever the Controller would do when no querystring provided)